apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -60,9 +64,23 @@ export default class VirtualizedDetailView extends Component {
         {(rowIndex === 0 && columnIndex === 0) && (
           <span style={styles.cell}>Lesson</span>
         )}
+        {(rowIndex === 0 && columnIndex >= 1) && (
+          <span style={styles.cell}>{columnIndex}</span>
+        )}
         {(rowIndex === 1 && columnIndex === 0) && (
           <span style={styles.cell}>Level Type</span>
         )}
+        {(rowIndex === 1 && columnIndex >= 1) && (
+          <span style={styles.cell}>
```

Comment:
I think the DOM for this case is something we'll want to pull into a separate component so we can have a storybook for it with "tested" data.

Reply:
I think this will probably be the case for most of the other cells too - definitely the ones that render the ProgressBubbleSet.  What do you think if we get everything "working" with real data, then extract out sub-components?

Reply:
Fine by me as long as it happens by end the of our "prototype phase"/this sprint. I don't want us to get too far with a structure that will make it confusing long term.